pkg/sched/sched_trace.go

```diff
@@ -50,20 +53,20 @@ func NewSchedTrace() *SchedTrace {
 // scheduler trace.
 func (st *SchedTrace) UpdateSchedTraceFromRawTrace(rawTrace string) {
 	split := strings.Split(rawTrace, " ")
-
 	// this is a lot of hacky code to get the information out of a scheduler trace
 	// line that looks as follows with GOMAXPROCS being 8:
-	// SCHED 0ms: gomaxprocs=8 idleprocs=6 threads=5 spinningthreads=1 idlethreads=2 runqueue=0 [0 0 0 0 0 0 0 0]
+	// SCHED 0ms: gomaxprocs=8 idleprocs=6 threads=5 spinningthreads=1 needspinning=1 idlethreads=2 runqueue=0 [0 0 0 0 0 0 0 0]
```

Comment:
Can we set `st.Gomaxprocs` in the constructor itself and avoid updating it at every tick? This is based off the assumption that we don't change the value of GOMAXPROCS through the run of this program, which seems reasonable for now.

Reply:
Done